members-only/.stickler.yml

```diff
@@ -0,0 +1,13 @@
+linters:
+  rubocop:
+    display_cop_names: true
+    fixer: true
+fixers:
+    enable: true
+files:
+  ignore:
+    - "bin/*"
+    - "db/*"
+    - "config/*"
+    - "Guardfile"
+    - "Rakefile"
```

Comment:
Let's add this stickler config file to the root of the project. Currently, stickler is not checking this PR.